src/Plugins/ComplexCore/src/ComplexCore/Filters/FindNeighborListStatistics.cpp

```diff
@@ -16,18 +17,18 @@ namespace
 {
 constexpr int64 k_NoAction = -6800;
 constexpr int64 k_MissingInputArray = -6801;
-constexpr int64 k_WrongInputArrayType = -6802;
-constexpr int64 k_NonScalarInputArray = -6803;
+constexpr int64 k_BoolTypeNeighborList = -6802;
+constexpr int64 k_EmptyNeighborList = -6803;
 
 template <typename T>
 class FindNeighborListStatisticsImpl
 {
 public:
   using NeighborListType = NeighborList<T>;
 
-  FindNeighborListStatisticsImpl(const IFilter* filter, INeighborList& source, bool length, bool min, bool max, bool mean, bool median, bool stdDeviation, bool summation,
+  FindNeighborListStatisticsImpl(const IFilter* thisFilter, INeighborList& source, bool length, bool min, bool max, bool mean, bool median, bool stdDeviation, bool summation,
```

Comment:
what is the reasoning for changing "filter" to "thisFilter"?

Reply:
was done for clarity when developing parallel method reverted now